src/lib.rs

```diff
@@ -119,7 +119,7 @@
 //!
 //! ## OPTIONS
 //!
-//! -c --config-dir &lt;config_dir&gt;
+//! -c --config-dir &lt;config dir&gt;
```

Comment:
No, this reflects the manpage and is not changed apart from pure formatting to make it work.